flixel/tile/FlxTilemap.hx

```diff
@@ -48,13 +51,6 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 	 */
 	private static var _helperBuffer:FlxTilemapBuffer = Type.createEmptyInstance(FlxTilemapBuffer);
 	
-	// TODO: remove this hack and add docs about how to avoid tearing problem by preparing assets and some code...
-	/**
-	 * Try to eliminate 1 px gap between tiles in tile render mode by increasing tile scale, 
-	 * so the tile will look one pixel wider than it is.
-	 */
-	public var useScaleHack:Bool = true;
```

Comment:
I agree with removing this, but it doesn't look like you've solved the problem with tile seams yet. The "TODO" above also glosses over the problem - if I'm using non-integer scaling, there's no way to prepare my assets to avoid tearing. Alpha bleed and tile gaps caused by the renderer are two different issues.

I had a pair of HaxePunk commits to fix the gap problem a few years ago. The idea was to always increment the draw position by an integer when moving to the next row or column, which would sometimes result in 1px difference in tile width/height but no gaps: https://github.com/HaxePunk/HaxePunk/commit/6622764cf0bce9fb568abffd0e0837f055ac7baa 
https://github.com/HaxePunk/HaxePunk/commit/2b8c81d1f42b5860a033415ed3ed6795942af261